card-data/03-encounters.yaml

```
@@ -0,0 +1,564 @@
+      - Be weakened or repulsed by warding rituals
+      - Make a compromise to achieve its objective
+- front: &Tense
+    <<: *tfront
+    name: Tense
+    desc: >
+      //"Deep breaths. Stay calm. Show them we're friends." -- Tana
+
+      A Tense being is one who is willing to interact peacefully,
+      but is ready to fight or flee.
+
+      Tense people can be negotiated with, but carefully.
+      They're on alert for a reason. What is it?
+      Other kinds of creatures can be approached,
+      maybe offered something they want like food, but they will be wary.
+
+      Keeping a Tense being from bolting or attacking may pose a challenge.
+
+      Tense beings may:
+    prompts:
+      - Demonstrate fear, anger, or mistrust
+      - Compel a concession or compromise from the group
+      - Distract someone from an impending danger
+  back:
+    <<: *tback
+    name: Tense
+    desc: >
+      //"You mean the real villain all this time was...?!" -- Emory
+      
+      Tension is resolved in good or bad ways.
+      Either way, it's possible to glimpse the reason for the tension.
+      
+      Perhaps a mother dragon was guarding her nest.
+      Perhaps the strange peddler was the King's spy.
+      Whatever is really going on,
+      you might never have learned about it if not for this encounter.
+      
+      Tense beings may:
+    prompts:
+      - Reveal a deceit or misunderstanding
+      - Build or break trust
+- front: &Venomous
+    <<: *tfront
+    name: Venomous
+    desc: >
+      //"Don't let it touch you! Your skin will rot away!" -- Woody
+
+      The being has a toxic quality that makes it dangerous to engage with.
+
+      Frogs and toads, insects, spiders, snakes, scorpions,
+      and beings based on them, may all be naturally deadly.
+      Dishonorable fighters such as assassins might make use of poisons,
+      usually on bladed weapons.
+
+      Avoiding harm while fighting a Venomous being may pose a challenge.
+
+      Venomous beings may:
+    prompts:
+      - Sicken anyone in contact with them
+      - Fend off an attack with the threat of toxin
+      - Deliberately attack with the poison
+  back:
+    <<: *tback
+    name: Venomous
+    desc: >
+      //"My arrows can fly farther than its spit." -- Woody
+
+      Poison is not selective. It will harm anyone, friend or foe.
+
+      Some poisons have antidotes, or may not penetrate some kinds of defense.
+      For example, a snake's fangs might be deflected by armor.
+
+      Being Venomous does not pose a challenge
+      if the source or effectiveness of the poison is negated.
+
+      Venomous beings may:
+    prompts:
+      - Inadvertently harm allies, or themselves if they are not immune
+      - Lose the harmful quality if it's concentrated in fangs, weapons, etc. that are lost
+      - Fail to harm something or someone with a natural or acquired immunity
+- front: &Zombie
+    <<: *tfront
+    name: Zombie
```

```suggestion
- front: &Undead
    <<: *tfront
    name: Undead
```
I know you probably chose the more colorful title very intentionally, but I think that there are too many strong and specific associations with zombies these days to prompt more general contributions.